actions/drain_node_handler_test.go

```diff
@@ -331,6 +331,24 @@ func TestGetDrainTimeout(t *testing.T) {
 	})
 }
 
+func TestLogCastPodsToEvict(t *testing.T) {
```

Comment:
The code looks good to me. It's just tests look a bit meaningless here: there is no asserting that we logged what we had. More generally it is not even subject for our testing to assert the logs, so it is up to you if you want to keep this test ;)

Reply:
@andrejatcastai the test is jus making sure this method won't panic, I tried to include test cases for castware pods in the drain tests, it was flaky, tests failed randomly after adding a new pod to validate this single log, I decided to add a test, the assertion is basically passing.

Reply:
@andrejatcastai added tests with assertion